src/core/Akka.Remote.Tests/RemoteDeathWatchSpec.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Actor.Dsl;
+using Akka.Configuration;
+using Akka.Event;
+using Akka.TestKit;
+using Akka.TestKit.TestEvent;
+using Akka.Util;
+using Xunit;
+
+namespace Akka.Remote.Tests
+{
+    public class RemoteDeathWatchSpec : AkkaSpec
```

Comment:
Ported a missing spec to verify some remote `DeathWatch` scenarios that weren't previously covered by tests.